codegen-test/model/rest-json-extras.smithy

```diff
@@ -93,7 +95,8 @@ structure ExtraError {}
         body: "rawstring",
         params: { payload: "rawstring" },
         method: "POST",
-        protocol: "aws.protocols#restJson1"
+        protocol: "aws.protocols#restJson1",
+        appliesTo: "client",
```

Comment:
Why all these `appliesTo: "client"`s? Many of these should run / can be made to run correctly for the server implementation too.

Reply:
Removed the unnecessary ones, thanks